assets/javascripts/discourse-assign/controllers/group-assignments.js.es6

```diff
@@ -0,0 +1,6 @@
+import { inject as service } from "@ember/service";
+import Controller from "@ember/controller";
+export default Controller.extend({
+  router: service(),
+  queryParams: ["route_type"]
```

Comment:
What does this parameter do? Can we name it something more descriptive?

Reply:
I just removed it, as it wasn't useful